cl/search/management/commands/cl_dumpdata.py

```diff
@@ -0,0 +1,72 @@
+'''
+    Hopefully this makes slicing out production data simpler.
+'''
+import random
+from django.core import serializers
+from django.core.management.base import BaseCommand
+from cl.search.models import Docket, OpinionCluster, Opinion
+
+
+class Command(BaseCommand):
+    help = ('CL-specific data dumper for making fixtures from production')
+
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-n',
+            type=int,
+            default=10,
+            help='Number of items in the export'
+        )
+        parser.add_argument(
+            '--format',
+            type=str,
+            default='json',
+            help='Serialization format [json, xml, yaml]'
+        )
+
+    def handle(self, *args, **options):
+        n = options['n']
+        fmt = options['format']
+
+        self.stdout.write(
+            'Generating dump of up to %s randomly selected Opinions' % n
+        )
+        pk_qs = Opinion.objects.values_list('id', flat=True)
+        if pk_qs.count() < n:
+            n = pk_qs.count()
+        
+        pks = random.sample(
+            pk_qs.all(),
+            n
+        )
+        cluster_pks = OpinionCluster.objects.filter(sub_opinions__in=pks) \
+                                    .values_list('id', flat=True).all()
```

Comment:
The `.all()` you have here and in the other spots isn't doing anything, is it? Can you purge it?